plugin/src/utils/cache.ts

```diff
@@ -305,24 +343,27 @@ export const restoreHardCachedNodes = async ({ hardCachedNodes }) => {
       }
 
       // restore each node
-      await helpers.actions.createNode(remoteNode)
+      helpers.actions.createNode(remoteNode)
```

Comment:
I mentioned it in your comment but just adding a reminder here too that we should await this